examples/hooks/9-Payment-Element.js

```diff
@@ -0,0 +1,161 @@
+// This example shows you how to set up React Stripe.js and use Elements.
+// Learn how to accept a payment using the official Stripe docs.
+// https://stripe.com/docs/payments/accept-a-payment#web
+
+import React from 'react';
+import {loadStripe} from '@stripe/stripe-js';
+import {PaymentElement, Elements, useElements, useStripe} from '../../src';
+
+import '../styles/common.css';
+
+const CheckoutForm = () => {
+  const [status, setStatus] = React.useState();
+  const [loading, setLoading] = React.useState(false);
+  const stripe = useStripe();
+  const elements = useElements();
+
+  const handleSubmit = async (event) => {
+    // Block native form submission.
+    event.preventDefault();
+
+    if (!stripe || !elements) {
+      // Stripe.js has not loaded yet. Make sure to disable
+      // form submission until Stripe.js has loaded.
+      return;
+    }
+
+    setLoading(true);
+
+    stripe
+      .confirmPayment({
+        elements,
+        redirect: 'if_required',
+        confirmParams: {return_url: window.location.href},
+      })
+      .then((res) => {
+        setLoading(false);
+        if (res.error) {
+          console.error(res.error);
+          setStatus(res.error.message);
+        } else {
+          setStatus(res.paymentIntent.status);
+        }
+      });
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <PaymentElement />
+      <button type="submit" disabled={!stripe || loading}>
+        {loading ? 'Processing...' : 'Pay'}
+      </button>
+      {status && <p>{status}</p>}
+    </form>
+  );
+};
+
+const THEMES = ['stripe', 'flat', 'none'];
+
+const App = () => {
+  const [pk, setPK] = React.useState(
+    window.sessionStorage.getItem('react-stripe-js-pk') || ''
+  );
+  const [sk, setSK] = React.useState(
+    window.sessionStorage.getItem('react-stripe-js-sk') || ''
+  );
```

Comment:
I'm not sure it's obvious enough that you shouldn't have the secret key in the frontend at all here, or what parts of this example would need to change to make that the case. 

Since this might encourage insecure integrations and also because this example is so different than what a real Payment Element implementation looks like, I would prefer to omit the example altogether.

Perhaps we can link to https://stripe.com/docs/payments/quickstart instead, which has a React Stripe.js + Payment Element example.

Reply:
I agree with that! I'll update the example to accept a clientSecret + public key.